lib/fuse.c

```diff
@@ -4669,6 +4675,7 @@ static const struct fuse_opt fuse_lib_opts[] = {
 	FUSE_LIB_OPT("noforget",              remember, -1),
 	FUSE_LIB_OPT("remember=%u",           remember, 0),
 	FUSE_LIB_OPT("modules=%s",	      modules, 0),
+	FUSE_LIB_OPT("parallel_direct_write=%d",parallel_direct_writes, 0),
```

Comment:
Nit: There should be a space after the comma.

Reply:
done